frontends/ol-util/src/components/PageBanner.tsx

```diff
@@ -48,9 +48,13 @@ const imageWrapperHeight = css<ImgProps>`
   }
 `
 
+/**
+ * Prefer direct use of `BannerPage` component.
+ */
 const BannerContainer = styled.div`
   position: absolute;
   width: 100%;
+  z-index: -1;
```

Comment:
This restores the original value, which I inappropriately removed in https://github.com/mitodl/open-discussions/pull/3659/files#diff-81cd8ecef16a778b60f2e836d96769a87f5082a154eab9fc0f3d6f9bb577cbc6L54